library/models/library_res_partner.py

```diff
@@ -21,6 +28,21 @@ class LibraryResPartner(models.Model):
     )
     
     genres_ids = fields.Many2many(comodel_name='library.book.genre')
+    
+
+    
+    @api.depends('first_name', 'last_name')
+    def _compute_name(self):
```

Comment:
No te funciona así?
Recuerda hacer el for rec in self: No vaya a ser que se modifiquen varios first_name a la vez, en la vista lista, y de un error. Es poco probable, porque los nombres suelen ser unicos pero es una buena práctica tener for rec in self: en los metodos computados.